examples/gradle-v2/kotlin-multiplatform-example/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    kotlin("multiplatform") version "1.9.25"
+    id("org.jetbrains.dokka") version "2.0.20-SNAPSHOT"
+}
+
+group = "org.dokka.example"
+version = "1.0-SNAPSHOT"
+
+kotlin {
+    jvm()
+    linuxX64()
```

Comment:
it's better to add multiple targets here if you are willing to test those, because Dokka could have issues with intermediate sourceSets if there is single target (https://github.com/Kotlin/dokka/pull/3663)

Reply:
Sure, I think adding most targets here shouldn't be a problem.